app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <%= form.label :duplicable do %>
+          <%= form.check_box :duplicable %>
+          This stage can serve as a production template.
```

Comment:
So here's the deal:

1. At one point in time, I cloned a stage and ran with it, not realizing it was somebody just messing around.

2. That stage had `bundle -v foo bar blah` and so it just spit out the current version of bundler. Having a "gold master" stage would fix this problem. If you can come up with a better name, I'm all ears, this was the best thing @sandlerr and I came up with.